tests/test_nwm30_grid_loading.py

```diff
@@ -0,0 +1,55 @@
+
+from pathlib import Path
+import filecmp
+
+import pandas as pd
+import pytest
+
+from teehr.loading.nwm30.nwm_grid_data import fetch_and_format_nwm_grids
+from teehr.loading.nwm22.utils_nwm import build_zarr_references
+from teehr.loading.nwm22.const_nwm import (
+    NWM22_UNIT_LOOKUP,
+)
+
+TEST_DIR = Path("tests", "data", "nwm22")
+WEIGHTS_FILEPATH = Path(TEST_DIR, "onehuc10_weights.parquet")
+
+
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
@samlamont what's this for?

Reply:
hmm good question.  At one point I was getting a warning related to numpy but it no longer appears.  Maybe I had an issue with my venv.  Will remove these

Reply:
Actually I still see this when running `test_weight_generation.py` using pytest, although it doesn't seem to show up for other tests. Have you come across this?  Googling suggests upgrading numpy, but I'm on the latest version `1.26.2`. Guess I'll just ignore the warning for now

![image](https://github.com/RTIInternational/teehr/assets/7386333/2e4835f0-e187-4ddc-a276-58da855f61d8)